src/tls13.c

```diff
@@ -11917,6 +11924,38 @@ int wolfSSL_no_dhe_psk(WOLFSSL* ssl)
     return 0;
 }
 
+#ifdef HAVE_SUPPORTED_CURVES
```

Comment:
Should these also be wrapped with `!defined(NO_PSK)`?

Reply:
OK.
TLS 1.3 PSK is PSK and session tickets.
Added protection around fields and code.